cmd/container-use/version.go

```diff
@@ -20,25 +24,47 @@ func init() {
 			date = buildTime
 		}
 	}
+
+	versionCmd.Flags().BoolP("system", "s", false, "Show system information (Docker, Git versions)")
+	rootCmd.AddCommand(versionCmd)
 }
 
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print version information",
 	Long:  `Print the version, commit hash, and build date of the container-use binary.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Printf("container-use version %s\n", version)
+	RunE: func(cmd *cobra.Command, args []string) error {
+		showSystem, _ := cmd.Flags().GetBool("system")
+
+		// Always show basic version info
+		cmd.Printf("container-use version %s\n", version)
 		if commit != "unknown" {
-			fmt.Printf("commit: %s\n", commit)
+			cmd.Printf("commit: %s\n", commit)
 		}
 		if date != "unknown" {
-			fmt.Printf("built: %s\n", date)
+			cmd.Printf("built: %s\n", date)
 		}
-	},
-}
 
-func init() {
-	rootCmd.AddCommand(versionCmd)
+		// Show system info if requested
+		if showSystem {
+			cmd.Printf("\nSystem:\n")
+			cmd.Printf("  OS/Arch: %s/%s\n", runtime.GOOS, runtime.GOARCH)
+
+			if dockerVersion := getDockerVersion(cmd.Context()); dockerVersion != "" {
+				cmd.Printf("  Docker: %s\n", dockerVersion)
+			} else {
+				cmd.Printf("  Docker: not found\n")
+			}
```

Comment:
can we maybe also try to report if the daemon is actually running or the socket is available? fine to do in a follow up, but basically i wanna hedge against ppl who have other container runtimes and no docker cli symlink... unless maybe dagger actually requires the symlink, idk.